### ISSUE TITLE
[E053] Could not read config.cfg from /root/anaconda3/lib/python3.8/site-packages/resume_parser/degree/model/config.cfg

### ISSUE THREAD

**Issue by ranyaphat29** [OmkarPathak/pyresparser#50]
[E053] Could not read config.cfg from /root/anaconda3/lib/python3.8/site-packages/resume_parser/degree/model/config.cfg

**Comment by lironmo** [comment]
OSError: [E053] Could not read config.cfg from

**Comment by zeinramadan1** [comment]
Having the same issue, anyone managed to solve?

**Comment by yassine20011** [comment]
@zeinramadan1 did you find a solution?

**Comment by zeinramadan1** [comment]
> @zeinramadan1 did you find a solution?

yes. don't use a mac with an a1 chip, use venv not conda, and follow the solution by [Quppi] (https://github.com/Quppi) on here https://github.com/OmkarPathak/pyresparser/issues/46

**Comment by yassine20011** [comment]
Thank you a lot
